source/api_documentation/transactions_api/_intro.rst

```diff
@@ -59,7 +59,9 @@ The API takes the following optional query parameters:
     - Filters for the type of transaction. Valid inputs are Call, PostCallEvent, Sale, or Signal. Sale maps to the Reported Conversion type.
 
 In order to ensure that all transactions are returned when using the from= and to= date query parameters,
-you should store the last transaction id you have downloaded and pass it as the start_after_transaction_id to the next request.
+you should store the last transaction id you have downloaded and pass it as the start_after_transaction_id to the next request *so long as you do not change the date range*.
+Once you update the date range, the best practice is to omit the start_after_transaction_id param for the first request, then use the last transaction id you have downloaded until the date range is complete.
+This is because using from and to params sorts transactions in relation to the call start time, not the transaction time.
 Typical usage on the polling interval is to repeatedly call the API until no rows are returned, meaning you have downloaded all transactions.
 Please note, the "to" and "from" date range parameters are both necessary, providing only one or the other will not filter the results.
```

Comment:
Hmm maybe out of scope for this specific documentation update, but I don't feel like this line is accurate. I think providing `from` suffices because `to` will be automatically assume today's date.

Providing neither still works, but it'll just pull transactions from the 25 months ago, which I can understand why we wouldn't suggest as a best practice.

Reply:
Would be interesting to confirm but this line predates my changes. I say we leave as is for now.

Reply:
Agreed